### DISCUSSION TITLE
How to combine two pagecrops

### DISCUSSION THREAD

**Discussion by kathimohan** [jsvine/pdfplumber#775] [Q&A] [answered]
pa = page.crop((cl[column][0], cl[column][1], cl[column][2], ch[1]))                                        
                                                                           
pb = page.crop((cl[column][0], ch[3], cl[column][2], cl[column][3]))

I have to combine pa and pb and save as pdf, Is there any way to do it.

**Comment by jsvine** [comment, marked as answer]
Hi @kathimohan, and thanks for your interest in this library. Unfortunately for your goals, `pdfplumber` does include functionality for creating PDFs, only for extracting information from them. You might instead be interested in [`pikepdf`](https://github.com/pikepdf/pikepdf) or other PDF-creation tools listed [here](https://johannesfilter.com/python-and-pdf-a-review-of-existing-tools/).
